src/Perlang.Stdlib/Internal/Extensions/TypeExtensions.cs

```diff
@@ -1,7 +1,8 @@
 using System;
 using System.Numerics;
+using Perlang.Lang;
 
-namespace Perlang.Extensions
+namespace Perlang.Internal.Extensions
```

Comment:
As part of fixing this problem:

> `AsciiString` is currently defined in `Perlang.Common`, whereas it should rightfully be in `Perlang.Stdlib`. However, moving it there will cause some interesting dependency-related issues that remain to be resolved.

...we moved certain classes (like this one) out of `Perlang.Common` and into an "internal" namespace in `Perlang.Stdlib`. They really don't belong in the (facing) stdlib at all; in the future, I envision moving these to some `Perlang.CommonStdlib` assembly or similar. ("shared code between both Perlang parser and interpreter which should not be publicly exposed to user code")